projects/v2/src/app/store/sheet.state.ts

```diff
@@ -490,6 +491,7 @@ export class SheetState {
         allResults.map((res: ResponseData, index: number) => {
           for (const row of res.data) {
             row.organName = organsNames[index];
+            row.generalPublications = res.metaData['General Publication(s)'];
```

Comment:
This seems bad. There should be a better way to pass around the sheet metadata for individual node/row displays. Also, the metadata array should have the human strings translated to something more appropriate well before this. (ie metadata.publications instead of metadata['General Publication(s)']).